src/utils/debug.js

```diff
@@ -52,11 +52,11 @@ export function translateErrorForUser(error) {
   if (error instanceof CoreError) {
     if (error.code == ErrorCodes.INSUFFICIENT_FUNDS) {
       text = translate('ErrorCodes.CoreErrorInsufficientFunds');
-    }
-
-    if (error.code == ErrorCodes.SAFE_NOT_FOUND) {
+    } else if (error.code == ErrorCodes.SAFE_NOT_FOUND) {
       //  error: `Could not find Safe with address ${safeAddress}`,
       text = error.message;
+    } else {
+      text = `${error.message}. ${translate('ErrorCodes.GeneralErrorMessage')}`;
```

Comment:
Why? See line 62?

Reply:
There are more CoreError and we do not have case for every type.
I added one more case when there are invalid options for example here it will populate:
"safeAddress" is missing in options. Please try with different values or refresh page or try again later"

Reply:
But logically it will be caught in the last else either way no?

Reply:
Yes, but I am adding those types so we are more aware of them and have more control over what do we want to display for the user.

Reply:
I understand what @mikozet is trying to do here. @louilinn if the error is instance of CoreError, it will never enter the last `else` statement thus will be not logged.

What we could do to not dirt the code with so much if/else is just to initialize the variable `text` with a default value like this ```let text = `${error.message}. ${translate('ErrorCodes.GeneralErrorMessage')}`;``` and then delete both `else` statements with same declaration.

Reply:
Yes ok I see now x) 
Yeah that would be the better option but it's ok to leave also. But this is confusing code :p

Reply:
I liked @juanenrisley idea I corrected it